.github/workflows/terraform-plan-apply.yml

```diff
@@ -15,13 +15,51 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Validate YML
+      - name: Validate Trial Definition (YML)
         id: validate_definiton_yml
         env:
-          INPUT_SCHEMA: .ci/trial-json-schema.json
+          INPUT_SCHEMA: .ci/definition-schema.json
         run: |
+          set -o errexit
+          set -o pipefail
+          set -o nounset
+
           sudo npm install -g ajv-cli
-          find . -name definition.yml -exec ajv validate -s $INPUT_SCHEMA -d {} \;
+
+          export FILES=$(find . -name 'definition.yml')
+          for FILE in $(echo "$FILES" | sed "s/,/ /g")
+          do
+               ajv validate -s $INPUT_SCHEMA -d "$FILE"
+          done
```

Comment:
Why do you need to run a for loop? Isn't the "before" way with the find and "exec" enough?

Reply:
didn't work due to the exit code not being picked up